doc/list.rst

```diff
@@ -1324,9 +1324,16 @@ List of Available Rules
 
    Native type hints for functions should use the correct case.
 
-   Part of rule sets `@PhpCsFixer <./ruleSets/PhpCsFixer.rst>`_ `@Symfony <./ruleSets/Symfony.rst>`_
+   *warning deprecated*   Use ``native_type_declaration_casing`` instead.
 
    `Source PhpCsFixer\\Fixer\\Casing\\NativeFunctionTypeDeclarationCasingFixer <./../src/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixer.php>`_
+-  `native_type_declaration_casing <./rules/casing/native_type_declaration_casing.rst>`_
+
+   Native type hints should be used in the correct case.
```

Comment:
```suggestion
   Native type declarations should be used in the correct case.
```

Reply:
thanks, will update, I'll update the other one as well as we are touching the same files ; )

Reply:
Thank you, @SpacePossum - I have opened a pull request updating additional mentions in #7338!

Reply:
Thanks so much :+1:  :)